actions/sequester/Quest2GitHub/Models/GithubIssue.cs

```diff
@@ -205,6 +236,37 @@ public class GithubIssue
                          item.GetProperty("bodyHTML").GetString()
                        );
 
+        var projectData = issueNode.Descendent("projectItems", "nodes");
+        var storyPoints = new List<StoryPointSize>();
+        if (projectData.ValueKind is JsonValueKind.Array)
+        {
+            foreach (var projectItem in issueNode.Descendent("projectItems", "nodes").EnumerateArray())
+            {
+                if (projectItem.ValueKind == JsonValueKind.Object)
+                {
+                    // TODO: MAUI uses one sprint project per year, with a field for the month.
+                    // Modify the code to store the optional month in the tuple field.
+                    // Consider: Store YYYY, Month, Size as a threeple.
+                    var projectTitle = projectItem.Descendent("project", "title").GetString();
+                    // size may or may not have been set yet:
+                    var sizeNode = projectItem.Descendent("fieldValueByName", "name");
+
+                    var size = (sizeNode.ValueKind is JsonValueKind.String) ? sizeNode.GetString() : null;
+                    if ((projectTitle is not null) && (size is not null))
+                    {
+                        storyPoints.Add(new StoryPointSize(projectTitle, size));
+                    }
+                }
+            }
+        }
+        var closedEvent = issueNode.Descendent("timelineItems", "nodes").EnumerateArray()
```

Comment:
Hate to think of edge cases like this, but what if it's closed and then reopened? Wouldn't this check need to be ordered by descending date, and then check if the last one is closed?